### PR TITLE
CONCF-372 | Move WikiaMobile setup to DefaultSettings, fix initialization of i18n in WikiaMobile

### PR DESCRIPTION
**Pull request by vforge**
Ping @igor @hakubo 
See https://github.com/Wikia/config/pull/1024

### CODE REVIEW

File: includes/wikia/DefaultSettings.php [hunk 1]
```diff
@@ -630,6 +630,7 @@
  * Includes that are needed for Mercury to work
  */
 
+include_once( "$IP/extensions/wikia/WikiaMobile/WikiaMobile.setup.php" );
```
Comment (rogatty) [inline includes/wikia/DefaultSettings.php]: Not sure if this should be just below the comment above.

Comment (hakubo) [inline includes/wikia/DefaultSettings.php]: yup, sounds right

Comment (Warkot) [inline includes/wikia/DefaultSettings.php]: But doesn't Mercury rely on WikiaMobile to remove the inline styles from the markup?

Comment (rogatty) [inline includes/wikia/DefaultSettings.php]: @Warkot yes, but WikiaMobile doesn't rely on Mercury ;)

**Comment by rogatty** [comment]
LGTM

**Comment by mklucsarits** [comment]
LGTM

**Pull request merged**